script/publish

```diff
@@ -74,14 +78,14 @@ verbose('git fetch origin')
 verbose('git reset --hard origin/master')
 verbose('git clean -ffxd .')
 
-File.write(version_file, File.read(version_file).gsub(/return '[.0-9]+';/, "return '#{version_string}';"))
 File.write(changelog_file, File.read(changelog_file).gsub(/\[Unreleased\]/, "[#{version_string}] - #{Date.today}"))
 verbose("npm version #{version_string} --no-git-tag-version")
 
 verbose('git add -A')
 verbose("git commit -m 'Publish #{tag}'")
 verbose("git tag -d #{tag} || true")
 verbose("git tag #{tag}")
+verbose("git tag #{semvertag} -m \"Release #{semvertag}.\"")
```

Comment:
Why do we need multiple tags?

Reply:
I don't know what the `amazon-chime-sdk-js@1.19.14` tags are for. The `v1.19.14`-style ones are what the rest of the world uses to indicate semver releases, including `git describe`. All of this magic functionality comes for free with Git if we just use the right tag format!

I would be happy to kill the custom one, but I figure it exists for *something*.

Reply:
@aburnson Should we stop use this tag amazon-chime-sdk-js@#{version_string}?